slo_generator/backends/dynatrace.py

```diff
@@ -262,6 +310,7 @@ def request(self,
             response = req(url, headers=headers, json=post_data)
         else:
             response = req(url, headers=headers)
+            LOGGER.debug(f'Response: {response}')
```

Comment:
Let's always log the response, not only in the `else` cause - so that we can debug `put` / `post` methods also.